cmd/lncli/walletrpc_active.go

```diff
@@ -1447,6 +1447,9 @@ var importAccountCommand = cli.Command{
 	The address type can usually be inferred from the key's version, but may
 	be required for certain keys to map them into the proper scope.
 
+	If an account with the same name already exists (even with a different
+	key scope), it will return an error.
```

Comment:
nit: use passive form, for example "an error will be returned", since it's not 100% clear what "it" refers to.